src/app/main/i18n/Language.ts

```diff
@@ -1,5 +1,10 @@
 export enum Language {
   UA = 'ua',
   EN = 'en',
-  RU = 'ru',
+  RU = 'ru'
```

Comment:
Remove Ru language - we already removed it from select

Reply:
I really want to do it, but can't? It use in other places of code. 
![image](https://github.com/ita-social-projects/GreenCityClient/assets/60553157/b2414172-5e20-475f-ac8b-01b754f3fa2b)

Reply:
I think it can be new task for future - remove ru language from the project . So you can keep it for now

Reply:
Thanks for the answer!